javax-money/pom.xml

```diff
@@ -86,6 +86,19 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <inherited>true</inherited>
+                <configuration>
+                  <annotationProcessorPaths>
```

Comment:
As per some comments, this section should no longer be needed and may even be counter-productive.
Does not seem to make any difference in this case wrt failure.